api/client/lib.py

```diff
@@ -919,10 +919,10 @@ def get_descendant_regions(access_token, api_host, region_id,
     for member_id in region['contains']:
         member = lookup(access_token, api_host, 'regions', member_id)
         if (not include_historical and member['historical']):
-            pass
-        elif descendant_level == member['level']:
+            continue
+        if not descendant_level or descendant_level == member['level']:
```

Comment:
`descendant_level` is still a required positional param, so you'd need to explicitly set it to `None` or `False` or `0`.

Could give it a default value of `None` so we could do something like `get_descendant_regions(1215)`. Or is that not desired since it would be making the more expensive case the default?

Reply:
Added default. Note it was already there in the calling Client method: https://github.com/gro-intelligence/api-client/blob/7dc79b81e9d9311b2ec4ad17ea53b126c6672ea7/api/client/__init__.py#L53

Reply:
Also if you set a default in the lib, need to also set a default in Client, or it'll still throw the required positional argument error if trying to use it without descendant_level

Reply:
note that this is the more general low level function where you know your region ids and region level ids. 

for simplicity, users can use wrappers like get_provinces(<country_name>)